### PR TITLE
Landingpage 2.0

### PR DESCRIPTION
**Pull request by siku2**
### Abstract
This PR splits the long description found on [`/landingpage`](https://swiss-voice.ch/landingpage) into two parts: a short preview and the rest of the text. The rest of the text is teased at using a fading preview and can be read by pressing the "Weiterlesen" button which unveils the rest of the text. *Obviously* this transition is animated and it works on both Desktop and Mobile

#### Desktop
Preview Desktop
#### Mobile
Preview Mobile

### CODE REVIEW

File: server/swissvoice/static/css/landingpage.css [hunk 1]
```diff
@@ -1,92 +1,126 @@
...
+.teaser + .teaser-trigger::after {
+    content: "Weiterlesen";
+}
+
+:not(.teaser) + .teaser-trigger {
+    margin-top: 1rem;
+}
+
+:not(.teaser) + .teaser-trigger::after {
+    content: "Nicht mehr Weiterlesen?? Plz improve";
```
Comment (siku2) [inline server/swissvoice/static/css/landingpage.css]: We **might** want to improve this little text over here tho

Comment (thieme55) [reply server/swissvoice/static/css/landingpage.css]: maybe: "verkürzen" or "vermindern" or "verringern"
-->personally "verringern" is my fav.

**Review by thieme55 — COMMENTED** [review]
nothing to say... but why so many spaces :)

**Review by thieme55 — APPROVED** [review]
I approve.

**Pull request merged**